pod/perldelta.pod

```diff
@@ -362,7 +362,7 @@ XXX
 XXX Important bug fixes in the core language are summarized here.  Bug fixes in
 files in F<ext/> and F<lib/> are best summarized in L</Modules and Pragmata>.
 
-XXX Include references to GitHub issues and PRs as: [GH #12345] and the release
+XXX Include references to GitHub issues and PR's as: I<[GH #12345]> and the release
```

Comment:
Emphasize the GH part, add ' to PR to better indicate the plural

Reply:
Apostrophes indicate possessive, not plural

Reply:
People sometimes use apostrophes in weird situations to indicate plural; I'm neutral about the practice.  In this case, I have a slight preference to not add it.

I however would like other feedback about italicizing the GH references.  I don't think it's worth the bother.

Reply:
What about applying C<> instead?

Reply:
> Apostrophes indicate possessive, not plural

Hmm, could use `-` here instead, it'd at least help separate the abbreviation

Reply:
Normally these tags get turned into links in the final release perldelta. Adding any other formatting here may encourage people to add that formatting when writing the perldelta and confuse things, so I would also side on omitting the formatting here.

Reply:
> People sometimes use apostrophes in weird situations to indicate plural; I'm neutral about the practice. In this case, I have a slight preference to not add it.

FWIW I've noticed fonts in which it is very hard to see that the 's' at the end of a pluralized abbreviation is lower-case, so it reads as PRS. An additional indication that a plural was intended could reduce confusion in such situations.

As far as formal correctness is concerned, I think classically the plural of PR (pull request) would be PR (pull requests).

> What about applying C<> instead?

That sounds like an excellent alternative.